Jenkinsfile

```diff
@@ -36,7 +36,14 @@ pipeline {
 
         stage("Run conjur_variable unit tests") {
           steps {
-            sh './dev/test_unit.sh'
+            sh './dev/test_unit.sh -r'
```

Comment:
Our Dockerfiles install `ansible-core` 2.13.x by default - we should include testing against all supported versions for fully maintainability. This will require some changes to our Jenkinsfile. We should include some conditionals that run tests in the following pattern:

| *Scenario* | *Ansible version* |
|------------|-------------------|
| Jenkins build: `main` branch | Run tests against all maintained, stable `ansible-core` versions (2.10-2.13), and `devel` branch |
| Jenkins build: PR branch | Run tests against `ansible-core` v2.13 |
| Jenkins build: release branch | Run tests against all maintained, stable `ansible-core` versions (2.10-2.13)